addons/mail/static/src/core/common/discuss_app_model.js

```diff
@@ -42,8 +42,8 @@ export class DiscussApp extends Record {
         return super.insert(...arguments);
     }
 
-    /** @type {'mailbox'|'all'|'channel'|'chat'|'livechat'} */
-    activeTab = "all";
+    /** @type {'main'|'channel'|'chat'|'livechat'} */
+    activeTab = "main";
```

Comment:
Should you rename this in stable? I know 17.0 is recent but it's still a stable version

Reply:
17.0 is not really stable, and also the rename is necessary so that `mailbox` and `all` are the same, to solve the syncing issue I mentioned in the commit message